packages/tools/client-logger/app-insights-logger/src/test/index.ts

```diff
@@ -0,0 +1,14 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * Exports test utilities shared by tests in the package.
+ */
+
+/* eslint-disable no-restricted-syntax */
+
+export * from "./ClientUtilities";
```

Comment:
I don't think we need to export anything from `src/test`, we only use `ClientUtilities` in the test app.